cards/src/main/resources/cards/custom/group6/witch_doctor/secrets/secret_ghastly_visage.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Ghastly Visage",
+  "baseManaCost": 2,
+  "type": "SPELL",
+  "heroClass": "ROSE",
+  "rarity": "COMMON",
+  "description": "Secret: After your opponent plays a minion, return it to its owner's hand.",
```

Comment:
This might need some tests because the behaviour can interact wonkily with  openers